### PR TITLE
adding my art

### PR DESCRIPTION
**Pull request by KareemMoneeam**
Please add me to this list. @royranger

### CODE REVIEW

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Changes to `src/include.js` are missing.

File: include.js [hunk 1]
```diff
@@ -6275,6 +6275,13 @@ let cards = [
     author: 'Katana',
     githubLink: 'https://github.com/katanaji'
   }
```
Comment (sophiabrandt) [inline include.js:6277]: There is a syntax error here. You need to separate each JavaScript object (the stuff in the curly braces) with commas.
```js
},
{
```

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Please fix the syntax error.

File: include.js (hunk 1 above)
Comment (KareemMoneeam) [reply include.js]: Sure! it's my fault.
Thank you for your good effort ♥

**Pull request merged**